Lib/test/test_dis.py

```diff
@@ -1124,6 +1124,43 @@ def test_bug_46724(self):
         # Test that negative operargs are handled properly
         self.do_disassembly_test(bug46724, dis_bug46724)
 
+    def test_annotate_source_locations(self):
+        # See gh-135700
+        issue_135700 = "1\nx: int"
+        issue_135700_class = "class A:\n    1\n    x: int"
+
+        test_cases = [
+            ("module", compile(issue_135700, "<string>", "exec").co_consts[1]),
+            (
+                "class",
+                compile(ast.parse(issue_135700_class), "?", "exec")
+                .co_consts[0]
+                .co_consts[1],
+            ),
+        ]
+
+        for case_name, annotate_code in test_cases:
```

Comment:
assert here that `annotate_code` is a code object and that `annotate_code.co_name` is `'__annotate__'`.